.github/workflows/lint.yml

```diff
@@ -7,7 +7,7 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 1
-          ref: ${{ github.head_ref }}
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
Why this change?

Reply:
Because it was failing to checkout the PR's. It worked on my own repo. I couldn't really explain it since I'm pretty sure I've seen it working you rinitial PR. I found this line in the readme of the action, tried it and it worked. this I stuck with it. If you have a better solution I'm open for it.